### PR TITLE
Added Yip Yip the kobold bartender

### PR DESCRIPTION
**Pull request by PolterTzi**
## About the PR

Adds a kobold equivalent for Pun Pun.

## Why / Balance

WEH

## Media

image
image

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl:
- add: A kobold might show up in Pun Pun's place.

### CODE REVIEW

**Comment by Simyon264** [comment]
FYI: There is a freeze on new pets.
> New pets. While fluff is nice we have enough cargo-obtainable fluff animals and department mascots, to add new animals they must have something interesting like new AI behaviors or spawn in proc gen maps.

See #8524 for more info. This PR may be closed any time by a maintaner.

**Comment by Errant-4** [comment]
At this point it might make sense to create a second parent for the bartender animals to inherit the common components without having to redefine them every time

**Comment by VasilisThePikachu** [comment]
> FYI: There is a freeze on new pets.
> 
> > New pets. While fluff is nice we have enough cargo-obtainable fluff animals and department mascots, to add new animals they must have something interesting like new AI behaviors or spawn in proc gen maps.
> 
> See #8524 for more info. This PR may be closed any time by a maintaner.

It wont be closed but not merged any time soon.

**Comment by PolterTzi** [comment]
Ah, darn.
Oh well.

File: Resources/Prototypes/Entities/Markers/Spawners/mobs.yml [hunk 1]
```diff
@@ -539,7 +539,7 @@
       - MobSlimesPet
 
 - type: entity
-  name: Pun Pun Spawner
+  name: Bartender Animal Spawner
   id: SpawnMobMonkeyPunpun
```
Comment (VasilisThePikachu) [inline Resources/Prototypes/Entities/Markers/Spawners/mobs.yml:543]: I would suggest you rename this as well and add a migration to change the old id to the new one

https://github.com/space-wizards/space-station-14/blob/master/Resources/migration.yml

**Review by VasilisThePikachu — CHANGES_REQUESTED** [review]
Though I will leave one review

**Comment by TheShuEd** [comment]
I think it's fine and doesn't really fall under the freeze, since it's not adding a new ghost role, but randomizing the appearance of an existing one.

**Comment by VasilisThePikachu** [comment]
> I think it's fine and doesn't really fall under the freeze, since it's not adding a new ghost role, but randomizing the appearance of an existing one.

>New pets. While fluff is nice we have enough cargo-obtainable fluff animals and **department mascots**, to add new animals they must have something interesting like new AI behaviors or spawn in proc gen maps.

I would call this a new department mascot. And it does not add anything interesting

**Comment by Carmell0** [comment]
Suggestions:
 The name “nup nup” or “nip nip” would be funnier
Splash text saying “engage in a rivalry with pun pun”
Sometimes pun and yip spawn together. Sometimes both are at the bar, or maybe make it so yip spawns in the maints bar.

**Comment by CatMagic1** [comment]
I would argue that this is cool and yall are using TOO much logic to decide on funny lizard bartender

**Comment by UbaserB** [comment]
> Suggestions:
> The name “nup nup” or “nip nip” would be funnier
> Splash text saying “engage in a rivalry with pun pun”
> Sometimes pun and yip spawn together. Sometimes both are at the bar, or maybe make it so yip spawns in the maints bar.

i agree with this

**Comment by Hmeister-real** [comment]
no explanation, just closed? This is a sad day

**Comment by VasilisThePikachu** [comment]
> no explanation, just closed? This is a sad day

they got repo banned

**Comment by Hmeister-real** [comment]
> > no explanation, just closed? This is a sad day
> 
> they got repo banned

oh... what for?

**Comment by FairlySadPanda** [comment]
ok hear me out 
image

**Comment by deltanedas** [comment]
> ok hear me out 
> image
> 

april 1 pun pun replacement

**Comment by Xman0613** [comment]
> > no explanation, just closed? This is a sad day
> 
> they got repo banned

wait why did they get repo banned?

**Comment by Jezithyr** [comment]
Reopened since the repoban was in error. (The ban was made due to a misunderstood reference to the stonewall riots in an april fools PR)

**Comment by gitjubx** [comment]
> > > no explanation, just closed? This is a sad day
> > 
> > 
> > they got repo banned
> 
> wait why did they get repo banned?

he censort the LGBTQ capes and made a joke about it beeing funny to censoring it or such...then wiggled himself out so like punch then apologize and everything is fine!

**Comment by PolterTzi** [comment]
> > > > no explanation, just closed? This is a sad day
> > > 
> > > 
> > > they got repo banned
> > 
> > 
> > wait why did they get repo banned?
> 
> he censort the LGBTQ capes and made a joke about it beeing funny to censoring it or such...then wiggled himself out so like punch then apologize and everything is fine!

I censored them in the media preview of the april fools PR that was meant to add them to riot gear (since "pride was a riot" and all that).
I very much did not intend to censor them in game.
The censorship was admittedly not a good idea and that's on me.

**Comment by metalgearsloth** [comment]
https://github.com/space-wizards/space-station-14/pull/25982 is a better approach to this.

**Comment by PolterTzi** [comment]
I don't really agree that it's strictly better, but there's little point in me arguing when it's still frozen anyway.

**Pull request closed without merging**